### PR TITLE
Release v1.3.0

### PR DESCRIPTION
**Pull request by saintube**
## ChangeLogs

1. Release v1.3.0.
    1) Add blog for v1.3.0 release.
    2) Update versioned_docs for v1.3.
    3) Update Chinese docs for v1.3.
3. Add design doc for Node Prediction.

### CODE REVIEW

File: versioned_docs/version-v1.3/designs/nri-mode-resource-management.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+  - kubelet -- CRI Request -> CRI Runtime -- OCI Spec --> OCI compatible runtime -> containers  
+  &emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&searr; &emsp; &nearr;  
+  &emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;&emsp;Koordlet NRI plugin
+
+![nri.png](/img/nri.png)
+
+## Upgrade Strategy
+
+- Need to upgrade containerd to 1.7.0+ or CRIO to 1.25.0+
```
[the remark this answers is not in the document]
Comment (saintube) [reply versioned_docs/version-v1.3/designs/nri-mode-resource-management.md:149]: OK

File: blog/2023-08-16-release/index.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+通过 [v1.3.0 Release](https://github.com/koordinator-sh/koordinator/releases/tag/v1.3.0) 页面，可以看到更多包含在 v1.3.0 版本的新增功能。
+
+## 未来计划
+
+在接下来的版本中，Koordinator 目前规划了以下功能：
+
+- 硬件拓扑感知调度，综合考虑节点 CPU、内存、GPU 等多个资源维度的拓扑关系，在集群范围内进行调度优化。
+- 提供节点可分配资源的放大机制。
+- Network QoS 的支持。
```
Comment (zwzhang0107) [inline blog/2023-08-16-release/index.md:164]: 这个文章里先不提

File: blog/2023-08-16-release/index.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+![nri](/img/nri-proposal.png)
+
+关于 Koordinator 启用 NRI 的部署方式，请见[Installation - Enable NRI Mode Resource Management](/docs/installation#enable-nri-mode-resource-management)。
+
+### 节点画像和 Mid 资源超卖
+
+Koordinator 中将节点资源分为4种资源优先级模型 Prod、Mid、Batch 和 Free，低优先级资源可以复用高优先级已分配但未使用的物理资源，以提升物理资源利用率；同时，资源优先级越高，提供的资源也越稳定，例如 Batch 资源采用高优先级资源短期（short-term）已分配但未使用的超卖资源，而 Mid 资源采用高优先级资源长周期（long-term）已分配但未使用的超卖资源。不同资源优先级模型如下图所示：
+
+![resource-priority-model](/img/resource-model.png)
```
Comment (zwzhang0107) [inline blog/2023-08-16-release/index.md:119]: 公式在这也贴一下

File: blog/2023-08-16-release/index.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+4. 重调度
+
+   - LowNodeLoad 支持按节点池设置不同的负载水位和参数等。自动兼容原有配置。
+   - 跳过 schedulerName 不是 koord-scheduler 的Pod，支持配置不同的 schedulerName。
+
+### NRI 资源管理模式
+
+Koordinator 的 runtime hooks 支持两种模式，standalone 和 CRI proxy，然而这两种模式各自有着一些限制。当前，尽管在 standalone 模式做了很多优化，但当想获得更加及时的 Pod 或容器的事件或者环境变量的注入时还是需要依赖 proxy 模式。然而， proxy 模式要求单独部署 koord-runtime-proxy 组件来代理 CRI (Container Runtime Interface) 请求, 同时需要更改 Kubelet 的启动参数并重启 Kubelet。
+
+NRI（Node Resource Interface），即节点资源接口，是 CRI 兼容的容器运行时插件扩展的通用框架，独立于具体的容器运行时（e.g. containerd, cri-o）, 提供不同生命周期事件的接口，允许用户在不修改容器运行时源代码的情况下添加自定义逻辑。特别的是，2.0 版本 NRI 只需要运行一个插件实例用于处理所有 NRI 事件和请求，容器运行时通过 Unix-Domain Socket 与插件通信，使用基于 Protobuf 的协议数据，和 1.0 版本 NRI 相比拥有更高的性能，能够实现有状态的 NRI 插件。
```
Comment (zwzhang0107) [inline blog/2023-08-16-release/index.md:107]: 提一下老docker的情况

Comment (saintube) [reply blog/2023-08-16-release/index.md:107]: added

File: blog/2023-08-16-release/index.md (hunk 3 above)
Comment (saintube) [reply blog/2023-08-16-release/index.md:121]: done

File: blog/2023-08-16-release/index.md (hunk 2 above)
Comment (saintube) [reply blog/2023-08-16-release/index.md]: fixed

**Review by eahydra — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**